arm-sql/2014-04-01/examples/DataMaskingPolicyCreateOrUpdate.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
what is the difference  between this file and " arm-sql/2014-04-01/examples/DataMaskingPolicyCreateOrUpdate.json"?